### PR TITLE
Addition of InMobi Advanced Bidding Adapters

### PR DESCRIPTION
**Pull request by RajSuvariya-InMobi**

### CODE REVIEW

File: InMobi/build.gradle [hunk 1]
```diff
@@ -0,0 +1,20 @@
+project.version = '9.1.1.0'
+project.ext.networkName = 'inmobi'
+
+apply from: '../shared-build.gradle'
+apply plugin: 'kotlin-android'
+
+dependencies {
+    implementation "com.inmobi.monetization:inmobi-ads:${sdkVersion}"
+    implementation "androidx.core:core-ktx:+"
```
Comment (im-lakshya) [inline InMobi/build.gradle:9]: Use specific version.

File: InMobi/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mopub.mobileads.inmobi">
```
Comment (im-lakshya) [inline InMobi/src/main/AndroidManifest.xml:3]: Check once, this should not be the same as the existing `CustomEventClass` that we ship

File: InMobi/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+## Changelog
+  * Initial Commit
```
Comment (im-lakshya) [inline InMobi/CHANGELOG.md:2]: Add something meaningful.

File: InMobi/src/main/AndroidManifest.xml (hunk 2 above)
Comment (RajSuvariya-InMobi) [reply InMobi/src/main/AndroidManifest.xml:3]: This is a hard guideline by MoPub anyway if both the plugins have the same package name then there won't be any issues as we have used different class names than the previous plugin.

Comment (im-lakshya) [reply InMobi/src/main/AndroidManifest.xml:3]: No its not about the class names, it about the `BuildConfig` file which is generated by the Android Library that is created. `BuildConfig` classes should not have any conflict.

**Review by im-lakshya — APPROVED** [review]

**Pull request merged**